content/en/opentelemetry/agent/_index.md

```diff
@@ -7,9 +7,34 @@ private: true
   The Datadog Agent with embedded OpenTelemetry Collector is in private beta.
 {{< /callout >}} 
 
-## Guides
+## Overview
 
-{{< whatsnext desc=" " >}}
-    {{< nextlink href="/opentelemetry/agent/install_agent_with_collector" >}}Install the Datadog Agent with the OpenTelemetry Collector{{< /nextlink >}}
-    {{< nextlink href="/opentelemetry/agent/agent_with_custom_components" >}}Use Custom OpenTelemetry Components with Datadog Agent{{< /nextlink >}}
-{{< /whatsnext >}}
\ No newline at end of file
+The Datadog Agent with embedded OpenTelemetry Collector is an open source Collector distribution that combines the flexibility of OpenTelemetry with the comprehensive observability capabilities of Datadog. This integrated solution includes:
+
+- Built-in Datadog pipelines and extensions
+- Support for traces, metrics, and logs
+- A curated set of components for optimal performance with Datadog
+
+{{< img src="/opentelemetry/embedded_collector/architecture.png" alt="Architecture overview for Collector embedded in the Datadog Agent." style="width:100%;" >}}
+
+## Key benefits
```

Comment:
Can we please rewrite this section to be aligned with overall positioning and messaging? see blog post on how we talk about it: https://www.datadoghq.com/blog/datadog-agent-with-otel-collector/

- Access Datadog’s comprehensive observability solutions (e.g. [750+ integrations](https://www.datadoghq.com/product/platform/integrations/) , [Live Container Monitoring](https://www.datadoghq.com/product/container-monitoring/), [Network Performance Monitoring](https://www.datadoghq.com/product/network-monitoring/network-performance-monitoring/) and [Universal Service Monitoring](https://www.datadoghq.com/product/universal-service-monitoring/) (via eBPF) etc.)  alongside existing OpenTelemetry capabilities via the embedded OTel Receivers, Processors, Exporters etc. 

_Context for the OTel capabilities if we want to elaborate: Simultaneously, you can directly leverage the growing number of OTel community-contributed integrations to collect telemetry in OTLP native format (via OTel receivers), and exercise control over your OTLP data with full access to the Collector’s powerful processing (via OTel processors) and routing (via OTel exporters) capabilities. With both, you get interoperability across your observability stack, whether open source or vendor-based._

- Manage fleets of embedded OpenTelemetry Collectors remotely with [Datadog Fleet Automation](https://docs.datadoghq.com/agent/fleet_automation/) and onboard faster without manual configurations such as out-of-the-box data enrichment with universal service tagging and Kubernetes tag extraction for your OTLP data.

- Receive dedicated assistance for onboarding and troubleshooting from Datadog’s global support team when using the Datadog Agent with the embedded OTel Collector and Datadog’s observability platform.